### ISSUE TITLE
Can't disable auto-indent when pasting

### ISSUE THREAD

**Issue by Blaisorblade** [syl20bnr/spacemacs#8897]
`SPC u p` should paste without indent, but fails since at least an year.

> Spc-u 4 p actually does paste four times without autoindention. Fun.

The relevant issue is #4219, but is mislabeled as question/forum/Python, which might be the reason it's open since a couple of years. There's even a proposed fix at https://github.com/syl20bnr/spacemacs/issues/4219#issuecomment-246074883.

**Comment by henriquenj** [comment]
I managed to reproduce this problem in the develop branch (3e98896816644cf8c09201b90d992f09fbe58d1e).

#### Description :octocat:

On master branch, `SPC u p` fails with `ad-Advice-evil-paste-after: Wrong type argument: number-or-marker-p, (4)`.

On develop branch, it will not fail, instead it will paste the same text 4 times as originally reported, but WITH indentation. This is easy to check on an C++ file, I basically copy some lines starting with "+" (I do this often when I'm copying from git diff buffers) and paste into a C++ buffer. With `p`, the indentation is performed and the pasted text is all deformed, but this is the expected bahaviour I guess. When using `SPC u p`, instead of the error, I get four copies of the yanked text, with the same deformation.

#### Reproduction guide :beetle:
- Start Emacs
- Copy the following text
```
+for (auto item : container)
+{
+    std::cout << "items: " << item << std::endl;
+}
```
- Paste into a C++ buffer using `SPC u p`

*Observed behaviour:* :eyes: :broken_heart:
The text will be pasted 4 times, with indentation.
![with_indent](https://user-images.githubusercontent.com/4898430/29490807-fd171694-851d-11e7-8113-ae2aeb9298f9.png)

*Expected behaviour:* :heart: :smile:
The text should be pasted one time, without indentation. This is what I get when adding C++ to `spacemacs-indent-sensitive-modes`.
![without_indent](https://user-images.githubusercontent.com/4898430/29490810-03a97a60-851e-11e7-8e5a-8c4e92bb76ca.png)

Analyzing the commands that are bound to `p` I get the following conclusions: on master branch, `p` is bound to `evil-paste-after` (which has specific help text stating how one could paste without indent by using `SPC u`); however on develop, the command has changed to `spacemacs/evil-mc-paste-after`, which does not say anything about universal argument, so I'm not sure if this new behaviour of pasting four times is a bug or not, at the very least I think it should be documented somehow.

#### Workaround
You can somehow work around this bug by modifying the variable `spacemacs-indent-sensitive-modes` on both develop and master. Here is an example to put into your `dotspacemacs/user-config` (replace with the modes you intent to paste without formatting)

```elisp
  ;; I want to disable pasting with formatting on C/C++ buffers
  (add-to-list 'spacemacs-indent-sensitive-modes 'c-mode)
  (add-to-list 'spacemacs-indent-sensitive-modes 'c++-mode)
```

#### System Info :computer:
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.200.9
- Spacemacs branch: develop (rev. 3e98896)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: ivy
- Layers:
```elisp
(ivy emacs-lisp git)
```

**Comment by brabalan** [comment]
I'm also on branch develop (81a931f48bdd6bcca4fdf0a3baab7b4bd7695591) and I have the same issue with both `SPC u p` and `SPC u P`.

**Comment by xuhdev** [comment]
Is this still relevant? I cannot reproduce this issue.

**Comment by duianto** [comment]
Thanks for testing this, your probably right.

This PR has been merged:
Check for universal argument before paste #11340

It superseeded this PR:
Fix pasting with universal argument #11858
which is marked as fixing this issue.
